src/App.js

```diff
@@ -13,7 +14,7 @@ class App extends Component {
       <div className="App">
         <Header />
         <main>
-          <CreatureList/>
+          <CreatureList creatures={creatures}/>
```

Comment:
sometimes I like to use the word 'prop' to help differentiate things. ex
```
<CreatureList creaturesProp={creatures}/>
```
You would need to reflect the name change in your CreatureList.js 
```
const creatures = this.props.creaturesProp
```
If you understand the concept, disregard but just something I wanted to bring up to help clarify if this doesn't make complete sense yet.